nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -1898,6 +1898,12 @@ public static String convToInternal(String name) {
             return KEY_FULLTEXT_BINARY;
         case NXQL.ECM_ACL:
             return KEY_ACP;
+        case DBSDocument.SUFFIX_MAJOR_VERSION:
+        case DBSDocument.UID_MAJOR_VERSION:
+            return DBSDocument.KEY_MAJOR_VERSION;
```

Comment:
You can `import static` the various `DBSDocument` constants to follow the style of the others.